src/Toml/Parser.hs

```diff
@@ -89,10 +89,11 @@ quote q t = q <> t <> q
 nonControlChar :: Parser Text
 nonControlChar = Text.singleton <$> satisfy (not . isControl)
 
-multilineStringP :: Parser Text
-multilineStringP = lexeme $ fmap mconcat $ quotes3
-                                        >> optional eol
-                                        >> allowedChar `manyTill` quotes3
+multilineP :: Parser Text -> Parser Text -> Parser [Text]
+multilineP quotes allowedChar = quotes >> optional eol >> allowedChar `manyTill` quotes
```

Comment:
Okay, let's introduce even more renaming. Let's add `P` suffix to all parsers, even arguments and parsers under `where`. So it will be `quotesP`, `allowedCharP`, `notControlCharP`, etc.